content/applications/inventory_and_mrp/manufacturing/management/using_work_centers.rst

```diff
@@ -0,0 +1,126 @@
+- :guilabel:`Cost per hour`: the cost of operating the work center for one hour
+- :guilabel:`Analytic Account`: the account where the cost of the work center should be recorded
+
+.. image:: using_work_centers/work-center-general-information.png
+   :align: center
+   :alt: The general information tab of the work center form.
+
+Assign equipment to a work center
+---------------------------------
+Using the :guilabel:`Equipment` tab, it is possible for specific pieces of equipment to be assigned
+to a work center. The following information will be displayed for each piece of equipment added:
+
+- :guilabel:`Equipment Name`: the name of the piece of equipment
+- :guilabel:`Technician`: the technician responsible for servicing the equipment
+- :guilabel:`Equipment Category`: the category the equipment belongs to
+- :guilabel:`MTBF`: mean time between failures; the average time that the piece of equipment will
+  operate before failing
+- :guilabel:`MTTR`: mean time to recovery; the average time it takes for the equipment to become
+  fully operational again
+- :guilabel:`Est. Next Failure`: an estimate of when the next equipment failure will occur
+
+.. image:: using_work_centers/work-center-equipment.png
+   :align: center
+   :alt: The equipment tab of the work center form.
+
+.. note::
+    :guilabel:`MTBF`, :guilabel:`MTTR`, and :guilabel:`Est. Next Failure` are all calculated
+    automatically based on past failure data, if any exists.
+
+Integrate IoT devices
+---------------------
+The :guilabel:`IoT Triggers` tab enables the integration of :abbr:`IoT (Internet of Things)` devices
+with a work center:
+
+- :guilabel:`Device`: specifies the IoT device to be triggered
+- :guilabel:`Key`: the security key for the device
+- :guilabel:`Action`: the IoT device action triggered
+
+.. image:: using_work_centers/work-center-iot.png
+   :align: center
+   :alt: The IoT Triggers tab of the work center form.
+
+Use case: configure an alternative work center
+==============================================
+When a work center is at capacity, it cannot accept any new work orders. Instead of waiting for the
+work center to become available, it is possible to specify an alternative work center where surplus
+work orders should be carried out.
+
+Begin by creating a new work center. Configure the :guilabel:`Equipment` tab so that it has all of
+the same equipment as the main work center. This will ensure that the same tasks can be carried out
+at both work centers. Navigate to the main work center and include the new work center in the
+:guilabel:`Alternative Workcenters` selection field.
+
+Now, create a new manufacturing order that uses the main work center for one of its operations. The
+main work center will automatically be selected for the operation in the :guilabel:`Work Orders`
+tab. After confirming the manufacturing order, click the :guilabel:`Plan` button that appears at the
+top left of the form.
```

Comment:
Added more instruction/context and separated the content with both line breaks and time-order language so it's easier to consume + pacing is easier to follow.

Now, if a reader came to this doc and didn't read the instructions before this section (which is very common since most doc reader skim), they should still be able to accomplish their desired outcome all the same 😉 

This section tbh is really a good example of what we're looking for, as it's narrative driven, and explains the UI _while_ giving instructions at the same time. Instead of List --> List --> List --> Use Case, every section like this could stand on it's own as an explicit, self-contained use case.

```suggestion
Begin by creating a new work center by navigating to :guilabel:`Manufacturing --> Configuration -->
Work Centers` and filling out the work center form, using the instructions above as a guide.

Next, fill out the :guilabel:`Equipment` tab so that it has all of the same equipment as the main
work center, which will ensure that the same tasks can be carried out at both work centers.

Then, navigate to the main work center and include the new work center in the :guilabel:`Alternative
Workcenters` selection field.

With the alternative work center configuration complete, create a new manufacturing order that uses
the main work center for one of its operations. The main work center will automatically be selected
for the operation in the :guilabel:`Work Orders` tab. After confirming the manufacturing order,
click the :guilabel:`Plan` button that appears at the top left of the form.
```